background-jobs/update-beach-club-rewards-function/src/entry-point.ts

```
@@ -0,0 +1,80 @@
+        console.log('✅ Migrations completed')
+        break
+
+      case 'reset':
+        console.log('⚠️  WARNING: This will delete all data!')
+        console.log('Press Ctrl+C to cancel, or wait 5 seconds to continue...')
+        await new Promise((resolve) => setTimeout(resolve, 5000))
+
+        const migrator = new (await import('./migrations/kysely-migrator')).KyselyMigrator(
+          db.rawPool,
+        )
+        await migrator.reset()
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)